tools/Dockerimage.crossbuild

```diff
@@ -0,0 +1,50 @@
+ARG AUTOWARE_DOCKER_ARCH
+ARG AUTOWARE_TARGET_ARCH
+FROM multiarch/alpine:${AUTOWARE_TARGET_ARCH}-latest-stable AS bootstrap
+
+FROM ${AUTOWARE_DOCKER_ARCH}/ros:dashing AS sysroot
+ARG AUTOWARE_TARGET_ARCH
+
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+
+COPY --from=bootstrap /usr/bin/qemu-${AUTOWARE_TARGET_ARCH}-static /usr/bin/qemu-${AUTOWARE_TARGET_ARCH}-static
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+  libboost-all-dev \
+  libtinyxml2-dev \
+  ros-dashing-osrf-testing-tools-cpp \
+  ros-dashing-ament-cmake-nose \
+  ros-dashing-launch-testing
+
+# hadolint ignore=SC2156
+RUN find / -depth -xdev -type l -lname '/*' -exec sh -c 'linkpath="$(readlink {})" && rm -f {} && ln -frsv "$linkpath" "{}"' \;
```

Comment:
Out of curiosity, what's the purpose of this line?

Reply:
This makes all links relative, so that they can be found when moved to the sysroot. For example, all the links in /etc/alternatives are absolute.

Reply:
I've added a comment explaining what this does